src/main/java/net/coderbot/iris/shaderpack/ShaderPack.java

```diff
@@ -241,4 +250,15 @@ public boolean isValid() {
 			}
 		}
 	}
+
+	static {
+		ShaderPack s;
+		try {
+			s = new ShaderPack(null);
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
Maybe provide some more context via a call to Iris's logger and the specific error?

Reply:
The try-catch is there simply because the constructor `throws IOException`. With the way I have things, an `IOException` should never actually be thrown from the `ShaderPack` constructor (and methods throwing `IOException` that are called in the constructor) when the passed Path is `null`. The try-catch is there due to the constructor's method signature already containing `throws IOException`.

Reply:
A likely change would be to just ignore the exception.